report/index.html

```diff
@@ -51,457 +52,479 @@
+            }
+
+            const pose = frame.getPose(component.space, primaryReferenceSpace);
+            if (pose) {
+              component.emulated = pose.emulated;
+              component.transform = pose.transform;
+            } else {
+              component.transform = null;
+            }
           }
         }
-      }
 
-      // Report if WebXR is supported at all.
-      let webXRSupportStatus = '❌ - Your browser does not support WebXR';
-      if ('xr' in navigator) {
-        webXRSupportStatus = '✔️ - Your browser supports WebXR'
-      } else if ('getVRDisplays' in navigator) {
-        webXRSupportStatus = '⚠️ - Your browser supports WebVR (deprecated), but not WebXR'
-      }
-
-      const modules = [
-        {
-          name: 'WebXR Device API (core)',
-          url: 'https://immersive-web.github.io/webxr/',
-          supported: 'xr' in navigator,
-        },
-        {
-          name: 'WebXR Gamepads',
-          url: 'https://immersive-web.github.io/webxr-gamepads-module/',
-          supported: 'gamepad' in XRInputSource.prototype
-        },
-        {
-          name: 'WebXR Augmented Reality',
-          url: 'https://immersive-web.github.io/webxr-ar-module/',
-          supported: 'environmentBlendMode' in XRSession.prototype
-        },
-        {
-          name: 'WebXR Hit Test',
-          url: 'https://immersive-web.github.io/hit-test/',
-          supported: 'requestHitTestSource' in XRSession.prototype
-        },
-        {
-          name: 'WebXR DOM Overlays',
-          url: 'https://immersive-web.github.io/dom-overlays/',
-          supported: 'domOverlayState' in XRSession.prototype
-        },
-        {
-          name: 'WebXR Layers',
-          url: 'https://immersive-web.github.io/layers/',
-          supported: 'XRProjectionLayer' in window
-        },
-        {
-          name: 'WebXR Anchors',
-          url: 'https://immersive-web.github.io/anchors/',
-          supported: 'createAnchor' in XRFrame.prototype
-        },
-        {
-          name: 'WebXR Lighting Estimation',
-          url: 'https://immersive-web.github.io/lighting-estimation/',
-          supported: 'requestLightProbe' in XRSession.prototype
+        // Report if WebXR is supported at all.
+        let webXRSupportStatus = '❌ - Your browser does not support WebXR';
+        if ('xr' in window.navigator) {
+          webXRSupportStatus = '✔️ - Your browser supports WebXR'
+        } else if ('getVRDisplays' in window.navigator) {
+          webXRSupportStatus = '⚠️ - Your browser supports WebVR (deprecated), but not WebXR'
         }
-      ];
-
-      const report = {
-        valid: false,
-        mode: null,
-        state: 'Invalid',
-        running: false,
-        session: null,
-        timestamp: 0,
-        referenceSpaces: {},
-        views: [],
-        inputs: [],
-        frame: null,
-        onEnd: () => {
-          if (report.session) {
-            report.session.end();
+
```

Comment:
This is the other substantive change, Firefox doesn't detect any of the bare WebXR API classes so it errors and prevent Vue from rendering the rest of the page properly. This allows the page to load properly.